daprdocs/content/en/js-sdk-docs/js-client/_index.md

```diff
@@ -37,11 +37,20 @@ const server = new DaprServer(serverHost, serverPort, daprHost, daprPort, Commun
 const client = new DaprClient(daprHost, daprPort, CommunicationProtocolEnum.GRPC);
 ```
 
-##### DaprClient Library
+{{% alert title="Note" color="primary" %}}  
+By default the example will run using HTTP. To use gRPC, do the following:
+
+- Replace both occurrences of `process.env.DAPR_HTTP_PORT` with `process.env.DAPR_GRPC_PORT`
+- Replace both occurrences of `CommunicationProtocolEnum.HTTP` with `CommunicationProtocolEnum.GRPC`
+- Instead of `npm run start:dapr-http` run `npm run start:dapr-grpc`
```

Comment:
`npm run start` is NPM specific and can be customized. Instead I would rather write `dapr run --app-protocol http` and `dapr run --app-protocol grpc` which is where we should change this. (Note that in Kubernetes this differs as well and it's in the `yaml` annotation

Reply:
So maybe we should add this clarification which makes it clear for people what this is changing?